plugins/woocommerce/includes/rest-api/Controllers/Version3/class-wc-rest-orders-controller.php

```diff
@@ -320,6 +321,12 @@ public function get_collection_params() {
 			),
 			'validate_callback' => 'rest_validate_request_arg',
 		);
+		
+		$params['manual_update'] = array(
+			'default'			=> false,
+			'description'		=> __( 'Set the action as manual so that the order note registers as "added by user".', 'woocommerce' ),
+			'type'				=> 'boolean'
+		);
```

Comment:
Sorry, I was wrong about where this arg definition needs to go. Putting it `collection_params` makes it show up as one of the args for _getting_ orders, but not for updating orders. Instead the definition needs to go in a separate method that looks like this:

```php
/**
 * Modify args for specific endpoints.
 *
 * @param string $method Optional. HTTP method of the request.
 *
 * @return array Endpoint arguments.
 */
public function get_endpoint_args_for_item_schema( $method = WP_REST_Server::CREATABLE ) {
	$args = parent::get_endpoint_args_for_item_schema( $method );

	if ( false !== stripos( WP_REST_Server::EDITABLE, $method ) ) {
		$args['manual_update'] = array(
			'default'     => false,
			'description' => __( 'Set the action as manual so that the order note registers as "added by user".', 'woocommerce' ),
			'type'        => 'boolean',
			'required'    => false,
		);
	}

	return $args;
}
```

Reply:
Actually, perhaps even better would be to add the definition to the `get_item_schema` method and set the context to edit, like this:

```php
$schema['properties']['manual_update'] = array(
	'default'     => false,
	'description' => __( 'Set the action as manual so that the order note registers as "added by user".', 'woocommerce' ),
	'type'        => 'boolean',
	'context'     => array( 'edit' ),
);
```

The `get_endpoint_args_for_item_schema` method ultimately pulls from `get_item_schema`, so this way we don't have to add an additional method.

Reply:
@coreymckrill I have added the definition to the `get_item_schema` method.